workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -118,24 +163,30 @@ def time_and_global_mean_value(diags):
     return time_and_global_mean_value
 
 
-@add_to_metrics("time_and_global_mean_bias")
-def time_and_global_mean_bias(diags):
-    time_mean_bias = grab_diag(diags, "time_mean_bias")
-    area = diags["area"]
-    time_and_global_mean_bias = weighted_mean(time_mean_bias, area, HORIZONTAL_DIMS)
-    restore_units(time_mean_bias, time_and_global_mean_bias)
-    return time_and_global_mean_bias
+for mask_type in ["global", "land", "sea"]:
 
+    @add_to_metrics(f"time_and_{mask_type}_mean_bias")
+    def time_and_domain_mean_bias(diags, mask_type=mask_type):
+        time_mean_bias = grab_diag(diags, f"time_mean_bias")
+        masked_area = _mask_array(mask_type, diags["area"], diags["land_sea_mask"])
+        time_and_domain_mean_bias = weighted_mean(
+            time_mean_bias, masked_area, HORIZONTAL_DIMS
+        )
+        restore_units(time_mean_bias, time_and_domain_mean_bias)
+        return time_and_domain_mean_bias
 
-@add_to_metrics("rmse_of_time_mean")
-def rmse_time_mean(diags):
-    time_mean_bias = grab_diag(diags, "time_mean_bias")
-    area = diags["area"]
-    rms_of_time_mean_bias = np.sqrt(
-        weighted_mean(time_mean_bias ** 2, area, HORIZONTAL_DIMS)
-    )
-    restore_units(time_mean_bias, rms_of_time_mean_bias)
-    return rms_of_time_mean_bias
+
+for mask_type in ["global", "land", "sea"]:
+
+    @add_to_metrics(f"rmse_of_time_mean_{mask_type}")
```

Comment:
This is a breaking change for the name of the "global" `rmse_of_time_mean` metric (which is used in the report maps' titles). Could we leave the name of the global one unchanged? E.g. don't add the mask type suffix for that case. Or alternatively, we could include the metric under both names and deprecate the one without the suffix for future removal.

Reply:
I'll go with the latter option and add a comment about deprecating it.